### ISSUE TITLE
Требуется открывать вручную инвентарь разгрузки для вложения вещей внутрь. (бесит)

### ISSUE THREAD

**Issue by hwat1204** [TauCetiStation/TauCetiClassic#12936]
#### Что должно было произойти

нажал по разгрузке предметов - тот положился в него!

#### Что произошло на самом деле

тебя имели, и нужно открывать вручную и ложить.

### Шаги воспроизведения

1: найти разгрузку (пример инженерной)

2: одеть

3: попытаться всунуть вещь!

**Comment by ZER0-In** [comment]
Такое случается, когда у тебя помимо разгрузки на униформу надето что-то еще. Тыкая в униформу ты тыкаешь в этот предмет. Крестик, фартук бармена или галстук. Все из-за неправильного порядка предметов на себе.

**Comment by ViDLiathan** [comment]
Да. На пустую одёжку надо первым делом разгрузку прицепить, а уже после всякие галстуки. И тогда будет нормально работать.
